CONTRIBUTING.md

```diff
@@ -194,3 +194,76 @@ IntelliJ may not pick up protoc generated classes as they can be very huge. If t
 
 ## CI
 The Ozone project uses Github Actions for its CI system.  The configuration is described in detail [here](.github/ci.md).
+
+## Debugging
+
+### Setup Ozone and Kubernetes
+
+Install a local server(k3s is recommended) and the Kubernetes command-line tool, kubectl. After setting up k3s and kubectl, build the project so that a SNAPSHOT image is created.
+
+1. Build the project `mvn clean install -DskipShade -DskipTests`
+2. `cd ozone/hadoop-ozone/dist/target/ozone-X.X..`
+3. Build the image `docker build -t apache/ozone:X.X-SNAPSHOT .`
+
+### Attach the debugger in IntelliJ
+
+Configure the debugger to attach to ports 5005, 6006, 7007.
+
+1. `Run -> Edit Configurations... -> Add New Configuration -> Remote JVM Debug`
+2. Name: `om-0`, Debugger mode: `Attach to remote JVM`, Host: `localhost`, Port: `5005`
+3. Name: `scm-0`, Debugger mode: `Attach to remote JVM`, Host: `localhost`, Port: `6006`
+4. Name: `s3g-0`, Debugger mode: `Attach to remote JVM`, Host: `localhost`, Port: `7007`
+
+Add the field `Command line arguments for remote JVM:` to running configurations so that it will be used as an argument
+when initializing ozone-manager, hdds-server-scm and ozone-s3gateway classes.
+
+In the file `ozone/hadoop-ozone/dist/target/ozone-X.X../bin/ozone.sh` 
```

Comment:
This should be:
In the file `<ozone-root>/hadoop-ozone/dist/target/ozone-X.X../bin/ozone`